routing_common/transit_types.hpp

```diff
@@ -153,6 +191,8 @@ class Edge
 
 class Transfer
 {
+  TRANSIT_TYPE_FRIENDS
```

Comment:
Please, move all TRANSIT_TYPE_FRIENDS to explicitly declared private section.

Reply:
Done